test/sqllogictest/retain_history.slt

```diff
@@ -99,7 +99,7 @@ idx_c  FOR  240000
 # Test subsource propagation. Test sources with and without subsources and view dependencies to
```

Comment:
changes in this file are a revert of https://github.com/MaterializeInc/materialize/commit/9bfaac448e71ce87a0e6d2ac8c6eacb4f3cde420 which seemed to break the intent of the test (to validate that the retain history values are propagated from source -> subsources / tables)